index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="styles/reset.css">
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="styles/footer.css">
+
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css" />
+</head>
+<body>
+    <header>
+        <div class="header-container">
+            <a href="index.html"><img src="/img/Property 1=md.png" alt="판다마켓 로고" width="130px">
+            </a>
+            <div class="login"><a href="login.html">로그인</a></div>
+        </div>
+    </header>
+
+    <main>
+        <section class="blue-top">
+            <div id="top" class="blue-container" >
+                <div class="text-box">
+                    <h1>일상의 모든 물건을 <br>거래해 보세요</h1>
+                    <div><a href="items.html">구경하러 가기</a></div>
```

Comment:
💊 제안
지금 작성하신 경우 현재 파일이 위치한 디렉토리 기준으로 이동됩니다. 예를 들어 www.example.com/ 이라면, www.example.com/items.html로 이동이 되고 www.example.com/privacy.html이라면 www.example.com/privacy.html/items.html을 찾게 됩니다.
상대경로보다는 절대 경로로 작성하시는 것이 이러한 문제나 유지보수 측면에서도 유리합니다~

```suggestion
                    <div><a href="/items.html">구경하러 가기</a></div>
```